index.html

```diff
@@ -20,11 +20,52 @@
 
 
 <body>
+    <nav>
+        <div class="menu-icons">
```

Comment:
- [ ] Good job implementing the hamburger button, but the mobile menu doesn't seem to appear, kindly make sure that the mobile menu appears after clicking the hamburger button